### PR TITLE
Add ids to recommended-payment & carriers & others

### PR DESCRIPTION
**Pull request by aleeks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.2.x
| Description?  | Add ids to recommended-payment & carriers & others
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | -
| How to test?  | -

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
* Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: admin-dev/themes/default/template/controllers/employees/helpers/form/form.tpl [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 {block name="field"}
 	{if $input.type == 'prestashop_addons'}
-		<div class="col-lg-{if isset($input.col)}{$input.col|intval}{else}9{/if} {if !isset($input.label)}col-lg-offset-3{/if}">
+		<div id="addons-connexion-container" ="col-lg-{if isset($input.col)}{$input.col|intval}{else}9{/if} {if !isset($input.label)}col-lg-offset-3{/if}">
```
Comment (hibatallahAouadni) [inline admin-dev/themes/default/template/controllers/employees/helpers/form/form.tpl]: @aleeks you delete the attribute class by mistake ;)

**Review by hibatallahAouadni — CHANGES_REQUESTED** [review]

File: admin-dev/themes/default/template/controllers/employees/helpers/form/form.tpl (hunk 1 above)
Comment (aleeks) [inline admin-dev/themes/default/template/controllers/employees/helpers/form/form.tpl]: thank you !

File: admin-dev/themes/default/template/controllers/employees/helpers/form/form.tpl [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
 {block name="field"}
 	{if $input.type == 'prestashop_addons'}
-		<div class="col-lg-{if isset($input.col)}{$input.col|intval}{else}9{/if} {if !isset($input.label)}col-lg-offset-3{/if}">
+		<div id="addons-connexion-container" class="col-lg-{if isset($input.col)}{$input.col|intval}{else}9{/if} {if !isset($input.label)}col-lg-offset-3{/if}">
```
Comment (eternoendless) [inline admin-dev/themes/default/template/controllers/employees/helpers/form/form.tpl]: `id="addons-connexion-container"`

If possible, please change "connexion" to "connection"

Comment (aleeks) [inline admin-dev/themes/default/template/controllers/employees/helpers/form/form.tpl]: ooooooooooooooh sh**t :( yes

**Comment by aleeks** [comment]
Done

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by eternoendless — APPROVED** [review]
Code looks good to me! Thanks @aleeks

**Comment by aleeks** [comment]
You can merge it btw! :smile:

**Pull request merged**